.github/workflows/test.yml

```diff
@@ -13,9 +13,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - php-version: 8.0
-          - php-version: 8.1
           - php-version: 8.2
+          - php-version: 8.3
```

Comment:
Might also do the compoesr flags thing here too.

Reply:
Good point!

Reply:
Done!